pkg/guacanalytics/patchPlanning.go

```diff
@@ -238,7 +247,7 @@ func exploreHasSourceAtFromSource(ctx context.Context, gqlClient graphql.Client,
 func explorePkgEqual(ctx context.Context, gqlClient graphql.Client, q *queueValues, pkgEqual model.NeighborsNeighborsPkgEqual) {
 	for _, pkg := range pkgEqual.Packages {
 		if pkg.Namespaces[0].Names[0].Versions[0].Id != q.now {
-			q.addNodeToQueue(PackageVersion, nil, pkg.Namespaces[0].Names[0].Versions[0].Id)
+			q.addNodeToQueue(PackageVersion, []string{pkg.Namespaces[0].Names[0].Versions[0].Id}, pkg.Namespaces[0].Names[0].Versions[0].Id)
```

Comment:
Is this supposed to be added? `[]string{pkg.Namespaces[0].Names[0].Versions[0].Id}` if so, it should be `.Version`

Reply:
Good catch, fixed.